data/gui/window/debug_clock.cfg

```diff
@@ -149,7 +149,7 @@
 						minimum_value = 0
 						maximum_value = 3599
 						step_size = 1
-						label = 3599
+						label = _ "3599"
```

Comment:
Should `[slider]label=` be translatable? It looks like it needs to be a number.

Reply:
I think it doesn't _need_ to be a number but usually will be… it also won't usually be set in the WML, since it needs to update dynamically as the slider moves.

This is the debug clock, so I don't really care if it's a bit weird, but I can change something if that seems like a good idea.